package.json

```diff
@@ -48,6 +48,7 @@
     "jsdom-global": "^3.0.2",
     "keycode": "^2.1.9",
     "lodash": "^4.17.4",
+    "material-ui": "0.18.1",
```

Comment:
Maybe it would be best to use `@material/dialog` like open-discussions does for consistency, @aliceriot what do you think?

Reply:
sounds good to me